KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -99,6 +99,12 @@ by the `linux` phase.
 Once you are done, either check the cluster manually, or use:  
 `export PHASES=k8s; export CHECK_CLUSTER=true; (cd cluster-provision/k8s/1.21; ../provision.sh)`
 
+### provision without caching images
```

Comment:
I'm not sure if the caching is immediately clear - we've called it `pre pulling` in kubevirtci provision.sh.
Maybe
```suggestion
### provision without pre-pulling images
```
Would be more self-descriptive?